projects/ng-boosted/src/lib/o-scroll/scroll-top.component.ts

```diff
@@ -15,9 +15,11 @@ export class ScrollTopComponent {
   public showMe = false;
 
   @Input() public label: string;
+  
+  @Input() public behavior: 'auto' | 'smooth' = 'auto';
 
   public ScrollToTop() {
-    window.scrollTo(0, 0);
+    window.scrollTo(0, 0, behavior);
```

Comment:
looks like it's not the way to pass behavior option to scrollTo function
Maybe try : scrollTo({ top:0, left:0, behavior: this.behavior }

Reply:
and it's not IE compliant, and iOS doesn't have smooth effect https://developer.mozilla.org/fr/docs/Web/API/ScrollToOptions

Need to check if it comply with accessibility rules too

Reply:
That is why i closed this MR, missing the This.

Reply:
You can update this PR and fix errors by pushing more commits to your branch